backend/lm_backend/api/models/feature.py

```diff
@@ -30,11 +32,27 @@ class Feature(Base):
     searchable_fields = [name]
     sortable_fields = [name, product_id, config_id]
 
+    @hybrid_property
+    def booked_total(self) -> int:
+        """
+        Compute the sum of all bookings.
+        """
+        return sum(b.quantity for b in self.bookings)
+
+    @booked_total.inplace.expression
+    @classmethod
+    def _booked_total_expression(cls) -> SQLColumnExpression[int]:
+        """
+        Provide a sql expression for computing the total bookings for each feature in a subquery.
+        """
+        return select(func.sum(Booking.quantity)).where(Booking.feature_id == cls.id).label("booked_total")
```

Comment:
Where is this query executed?

Reply:
This isn't being used currently, however, if you ever needed to use the booked_total inside of a query, then this would provide the sql logic automatically. Without it, the query expression wouldn't know how to use booked_total.

With hybrid attributes, it's common to supply an expression for instances where the hybrid attribute needs to be used in a query.

This was leftovers from when I was working on getting this working without using a lazy loader. However, the logic is sound and tested. I think it would be good to leave this in.